Assets/Scripts/Controllers/JobSpriteController.cs

```diff
@@ -68,7 +68,7 @@ void OnJobCreated(Job job)
             //until the graphics system for tiles is fleshed out further
 
             job_go.transform.position = new Vector3(job.tile.X, job.tile.Y, 0);
-            sr.sprite = SpriteManager.current.GetSprite("Tile", "Empty");
+            sr.sprite = SpriteManager.current.GetSprite("Tile", "Solid");
```

Comment:
I modified the Empty tile to be transparent with outlines, but I needed another tile to be a solid color to retain the green job tile.